webapp/components/survey/CategoryDetails/ImportSummary/TableRow.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+import * as Category from '@core/survey/category'
+import * as CategoryImportSummary from '@core/survey/categoryImportSummary'
+import * as Languages from '@core/app/languages'
+
+import { useI18n } from '@webapp/store/system'
+import Dropdown from '@webapp/components/form/Dropdown'
+
+const TableRow = ({ idx, columnName, column, setCategoryImportSummaryColumnDataType }) => {
```

Comment:
PropTypes is missing

Reply:
of course, I've just moved the component outside of another file

Reply:
would you add it?